deployments/charts/kuma-cp/values.yaml

```diff
@@ -0,0 +1,60 @@
+# The default registry and tag to use for all Kuma images
+global:
+  image:
+    registry: "kong-docker-kuma-docker.bintray.io"
+    tag: "0.6.0"
```

Comment:
Can this refer to AppVersion in the Chart

Reply:
We can't set that here in the values.yaml file since it's not templated, but can in the _helpers.tpl, lemme play around with that.

Reply:
I think this will work:
```golang
{{- define "kuma-cp.formatImage" -}}
{{- $img := .image }}
{{- $root := .root }}
{{- $registry := ($img.registry | default $root.Values.global.image.registry) -}}
{{- $repo := ($img.repository | required "Must specify image repository") -}}
{{- $defaultTag := ($root.Values.global.image.tag | default .Chart.AppVersion) -}}
{{- $tag := ($img.tag | default $defaultTag) -}}
{{- printf "%s/%s:%s" $registry $repo $tag -}}
{{- end -}}
```